invenio_cli/helpers/cli_config.py

```diff
@@ -227,11 +228,17 @@ def write(cls, project_dir, flavour, replay):
         config_parser[cls.CLI_SECTION] = {}
         config_parser[cls.CLI_SECTION]["flavour"] = flavour
         config_parser[cls.CLI_SECTION]["logfile"] = "/logs/invenio-cli.log"
-
+        custom_package_managers = {
+            "python_package_manager": "uv",
+            "javascript_package_manager": "pnpm",
```

Comment:
```suggestion
            "javascript_package_manager": replay.get("javascript_package_manager", None) or self.javascript_package_manager.name,
```
note: this is just from the top of my head, i doubt that it would work as is

from a quick check, it looks like this code path is only really executed in `invenio-cli init`, so accessing the replay would make a lot of sense i think
otherwise (if no replay is available... idk, would it be `None`?) we could fall back to the auto-detection logic above (which most likely defaults back to `npm` because we don't have any real detection logic there yet)

personally i would omit the `python_package_manager` as the auto-detection logic there should be fine unless you have both a `Pipfile` *and* `pyproject.toml` in your directory

Reply:
Resolved after a DM on Discord.